app/backend/approaches/chatreadretrieveread.py

```diff
@@ -30,26 +30,26 @@ class ChatReadRetrieveReadApproach(Approach):
     Try not to repeat questions that have already been asked.
     Only generate questions and do not generate any text before or after the questions, such as 'Next Questions'"""
 
-    query_prompt_template = """Below is a history of the conversation so far, and a new question asked by the user that needs to be answered by searching in a knowledge base about employee healthcare plans and the employee handbook.
-    Generate a search query based on the conversation and the new question. 
+    query_prompt_template = """Assistant is intelligent bot that helps to generate a search query.
+    Below is a history of the conversation so far, and a new question asked by the user that needs to be answered by searching in a knowledge base about employee healthcare plans and the employee handbook.
+    Generate a SEARCH QUERY based on the previous chat history and the new question. 
     Do not include cited source filenames and document names e.g info.txt or doc.pdf in the search query terms.
     Do not include any text inside [] or <<>> in the search query terms.
+    Do not include special characters like '+' extra
     If the question is not in English, translate the question to English before generating the search query.
 
-Chat History:
-{chat_history}
-
-Question:
-{question}
-
-Search query:
+Search Query:
 """
-
-    def __init__(self, search_client: SearchClient, chatgpt_deployment: str, chatgpt_model: str, gpt_deployment: str, sourcepage_field: str, content_field: str):
+    query_prompt_few_shots = [
+        {'role' : USER, 'content' : 'What is included in my Northwind Health Plus plan that is not in standard?' },
+        {'role' : ASSISTANT, 'content' : 'Northwind Health Plus plan benefits comparison standard' },
```

Comment:
If you're using the UI's sample questions for the few-shot, then you should change the sample question in the UI (as it seems a bit too easy for the bot!). Alternatively, you could come up with different example questions.

Reply:
Thanks for the catch. I have revised the shots .